### DISCUSSION TITLE
新添加存储的时候 提示：Failed create storage in database: database is locked

### DISCUSSION THREAD

**Discussion by imopro** [AlistGo/alist#3109] [General]
Alist 版本 V3.8-17

数据库:sqlite3

新添加存储的时候 提示：Failed create storage in database: database is locked

编辑已有存储时也提示这个错误信息。另外，创建用户时也提示：database is locked

这个数据库已锁的问题如何解决呢？

搜索可下，以前有人遇到过这个问题，但好像是v3.6下，然后这个问题被修正了。目前又出现了？

**Comment by imopro** [comment]
已解决。
systemctl restart 是不行。但先 stop 再start就好了。

**Comment by anwen-anyi** [comment]
database is locked 是你还在构建索引呢吧 因为sqlite3不支持并发 索引期间锁数据库了禁止其他操作

具体自己看自己理解：https://alist.nn.ci/zh/guide/advanced/search.html#搜索提示
